src/shared/ui/subtitle/subtitle.module.scss

```diff
@@ -0,0 +1,40 @@
+.subtitle {
+  margin: 0;
+  font-weight: $font-weight-medium;
+  line-height: 28px;
+  letter-spacing: 0;
+
+  &.font-medium {
+    font-size: 20px;
+
+    @include media-tablet {
+      font-size: 18px;
+      font-weight: $font-weight-regular;
+    }
+  }
+
+  &.font-large {
+    font-size: 24px;
+
+    @include media-tablet {
+      font-size: 18px;
+      font-weight: $font-weight-regular;
+    }
+  }
+
+  &.gray {
+    color: $color-gray-600;
+  }
+
+  &.black {
+    color: $color-black;
+  }
+
+  &.margin-small {
+    margin-bottom: 8px;
+  }
+
+  &.margin-medium {
+    margin-bottom: 16px;
+  }
+}
```

Comment:
We need a margin-bottom 4px for hero and main

Reply:
I guess we'd like to make the components more generic, so let's keep ]8px]